modules/api/src/main/java/org/eclipse/xpanse/api/controllers/ServiceCatalogApi.java

```diff
@@ -166,7 +167,13 @@ private UserOrderableServiceVo convertToUserOrderableServiceVo(
             userOrderableServiceVo.setDescription(
                     serviceTemplateEntity.getOcl().getDescription());
             userOrderableServiceVo.setBilling(serviceTemplateEntity.getOcl().getBilling());
-            userOrderableServiceVo.setFlavors(serviceTemplateEntity.getOcl().getFlavors());
+            List<FlavorBasic> flavorBasics = serviceTemplateEntity.getOcl().getFlavors()
+                    .stream().map(flavorWithProperties -> {
+                        FlavorBasic flavorBasic = new FlavorBasic();
+                        BeanUtils.copyProperties(flavorWithProperties, flavorBasic);
```

Comment:
@baixinsui flavorWithProperties must be just flavor now?

Reply:
Updated.